packages/clerk-js/src/ui/components/SignUp/SignUpContinue.tsx

```diff
@@ -150,6 +152,9 @@ function _SignUpContinue() {
     card.setLoading();
     card.setError(undefined);
 
+    const redirectUrl = buildSSOCallbackURL(ctx, displayConfig.signUpUrl);
```

Comment:
I see that we do this constantly, does it make sense to move it inside `useSignUpContext` ?